test/e2e/play_kube_test.go

```diff
@@ -5879,4 +5879,80 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 		Expect(inspect).Should(Exit(0))
 		Expect(inspect.OutputToString()).To(Equal(defaultUmask))
 	})
+
+	// podman play with infra name annotation
+	It("podman play kube test with infra name annotation set", func() {
+		infraName := "infra-ctr"
+		podName := "mypod"
+		outputFile := filepath.Join(podmanTest.TempDir, "pod.yaml")
+		pod := podmanTest.Podman([]string{"pod", "create", "--infra-name", infraName, podName})
+		pod.WaitWithDefaultTimeout()
+		Expect(pod).Should(Exit(0))
+
+		ctr := podmanTest.Podman([]string{"run", "-d", "--pod", podName, ALPINE, "top"})
```

Comment:
Nit: using `create` instead of `run` will make the test slightly faster.

Reply:
Same below.

Reply:
Fixed